app/models/room.rb

```diff
@@ -1,3 +1,9 @@
 class Room < ApplicationRecord
   belongs_to :hotel
+  has_many :guest_rooms
+  has_many :guests, through: :guest_rooms
+
+  def count_guests
+    guests.count
```

Comment:
Nice work encapsulating this logic. This could be enhanced with `distinct` so that we don't get duplicate names